src/state-calculator.tsx

```diff
@@ -238,7 +238,7 @@ const StateCalculator: FC<{
         <div className="text-color-text-secondary text-[0.75rem] leading-tight pb-[0.1875rem]">
           {msg('We’re dedicated to safeguarding your privacy.')}{' '}
           <a
-            className="text-color-action-primary font-medium"
+            className="text-color-text-link font-medium hover:underline"
```

Comment:
adding `hover:underline` as a drive-by; all the other link-looking things in the embed have that.